apps/api/src/hyp3_api/validation.py

```diff
@@ -77,21 +77,24 @@ def check_dem_coverage(granule_metadata):
 
 
 def check_same_burst_ids(granule_metadata):
-    ref_burst_id, sec_burst_id = [granule['name'].split('_')[1] for granule in granule_metadata]
-    if ref_burst_id != sec_burst_id:
+    burst_ids = [granule['name'].split('_')[1] for granule in granule_metadata]
+    not_matching = [burst_id for burst_id in set(burst_ids) if burst_ids.count(burst_id) == 1]
+    if not_matching:
```

Comment:
Would this implementation perhaps become simpler and easier to understand if the validator function accepted `reference` and `secondary` parameters?

Reply:
Yeah I think that would be better.

Reply:
Yes but all of the direct validator functions take granule_metadata because of validate_jobs. We could re-write validate_jobs and get_granules, but that would be at the cost of consistency across how we do validation.